docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md

```diff
@@ -53,6 +54,24 @@ This example also shows how to pass the administrative credentials for cluster C
 
 ## PARAMETERS
 
+### -AttemptSoftReboot
+Indicates that command assumes a Kernel Soft Reboot (KSR) for the failover cluster.
+
+KSR bypasses  BIOS/FIRMWARE initialization.
+You can only use KSR for updates that do not require a firmware/BIOS initialization.
```

Comment:
Can you make BIOS/FIRMWARE consistent?

Reply:
Found the article that I based this on and picked a consistent wording.